lib/AbstractConverter.php

```diff
@@ -17,7 +17,7 @@
     );
 
     protected $namespaces = array(
-        'http://www.w3.org/2001/XMLSchema' => '',
+        'http://www.w3.org/2001/XMLSchema'     => '',
```

Comment:
to be honest i never liked the arrays with the dynamic spacing... (it is not convenient if you add a new entry and the entry is longer the the previously longest)
Btw it is not PSR mandatory.

I prefer the current array initialization